### PR TITLE
Fix conflicts with integration tests

### PR DESCRIPTION
**Pull request by settinghead**
- isolate test environment for PlaceholderController & PlaylistItemController

### CODE REVIEW

**Comment by settinghead** [comment]
@Rise-Vision/apps This PRresolves conflict with integration tests. Specifically,tests for `PlaceholderController` and `PlaylistItemController` have been modified to run in isolation, with most of their dependencies mocked. The issue seems to be solved without further errors.

File: src-js/Viewer/Content/Content.integration.tests.js [hunk 1]
```diff
@@ -89,6 +86,22 @@ describe.skip("integration:", () => {
...
+      PlaylistItemInfo = require("../Info/PlaylistItemInfo");
+      ViewerDataInfo = require("../Data/ViewerDataInfo");
+      PresentationParser = require("../Parser/PresentationParser");
+    });
+    afterEach(() => {
+      if (require.cache) {
+        delete require.cache["./PlaylistItemController"];
+        delete require.cache["./PlaceholderController"];
+        delete require.cache["./ScheduleController"];
+        delete require.cache["./PresentationParser"];
```
Comment (alex-deaconu) [inline src-js/Viewer/Content/Content.integration.tests.js]: @settinghead not sure if it makes a difference, but you're missing `delete` for `ViewerDataInfo` & `PlaylistItemInfo`. Thanks.

**Comment by alex-deaconu** [comment]
@settinghead looks good. Just a couple of clarifications inline. Thanks.

File: src-js/Viewer/Content/Content.integration.tests.js (hunk 1 above)
Comment (settinghead) [inline src-js/Viewer/Content/Content.integration.tests.js]: @alex-deaconu probably not, but good to be rigorous here :). Will add them.

**Comment by alex-deaconu** [comment]
@settinghead looks good. Thanks for fixing 👍

**Pull request merged**